localization/english/wc_traits_l_english.yml

```diff
@@ -498,4 +498,8 @@
 
  trait_creature_nightborne:0 "Nightborne"
  trait_creature_nightborne_desc:0 "The Nightborne are a mystic race of elves originating from Suramar. Since the destruction of the Well of Eternity, the isolated Nightborne has evolved to become a new race thanks to the power of the Nightwell. Like other elves they require sustenance in the form of magic. Because of this the Nightborne employ the power of the Nightwell in the form of Arcwine to sustain themselves. The Nightborne remain the only elven race to maintain the legacy of the Kaldorei empire, sustaining the advancement of magic long after Azshara fell."
- trait_creature_nightborne_character_desc:0 "This character is a Nighborne elf. $trait_creature_nightborne_desc$"
\ No newline at end of file
+ trait_creature_nightborne_character_desc:0 "This character is a Nighborne elf. $trait_creature_nightborne_desc$"
+
+ trait_creature_drogbar:0 "Drogbar"
+ trait_creature_drogbar_desc:1 "The Drogbar are a race created by the Titans by modifying the Trogg. Formerly slaves to the black dragon Neltharion, the mostly subterranean race live largely at relative peace with their neighbors, the Highmountain Tauren. The Drogbar are adept in earth magic, and as such they use rocks for most purposes. With their magic rocks are carved to be used for furniture, containers, and even instruments like drums. They are even masters at Jewelcrafting; in particular the chief shaman of a tribe usually wears a neckless of heave gems."
```

Comment:
> the mostly subterranean race live largely at relative peace with their neighbors, the Highmountain Tauren

What if the drogbars migrated to Kalimdor and created the Drogbar Empire fighting the Great Triumvirate of Murloc Kingdoms.